docs/src/pages/docs/react/AsyncBoundary.en.mdx

```diff
@@ -38,7 +42,7 @@ const Example = () => (
 
 Since it wraps `<Suspense.CSROnly/>`, it provides the same CSROnly.
 
-```tsx
+```tsx /AsyncBoundary.CSROnly/
```

Comment:
<img width="603" alt="image" src="https://github.com/suspensive/react/assets/57122180/90a2ec4f-7d54-40d8-9a25-561a9353c769">

We need to update this end tag line